tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -107,6 +107,32 @@ def ldap_tear_down():
         ldap_auth.delete()
 
 
+@fixture()
+def groups_teardown():
+    "teardown for groups created for external/remote groups"
+    yield
+    # tier down groups
+    for group_name in ('sat_users', 'sat_admins'):
+        user_groups = entities.UserGroup().search(query={'search': f'name="{group_name}"'})
```

Comment:
Would it be possible to search for both group at the same time and just loop over results with `.delete()` call?

Reply:
Well, we could read all the groups and then loop over the `.delete()` call. 
So yes, it is possible. But then again, I think reading all the groups is too much work when we exactly know which groups to delete.

Reply:
I don't mean looping over all groups in Satellite, I mean searching for two groups you want at the same time, and then looping over results. Should be faster, as it would save you one `.search()` call.